docs/pages/guides/using-sentry.mdx

```diff
@@ -333,6 +338,143 @@ Sentry.init({
 
 </Collapsible>
 
+
+
+</Tab>
+
+<Tab label="SDK 50 beta">
+
+## Install and configure Sentry
+
+<Step label="1">
+
+### Sign up for a Sentry account and create a project
+
+Before proceeding with installing Sentry, you'll need to make sure you have created a Sentry account and project. Here's how to do that:
+
+<Step label="1.1">
+
+[Sign up for Sentry](https://sentry.io/signup/) (it has a free tier), and create a project in your
+Dashboard. Take note of your **organization slug**, **project name**, and **`DSN`** as you'll need
+them later:
+
+- **organization slug** is available in your **Organization settings** tab
+- **project name** is available in your project's **Settings** > **Projects** tab (find it in the list)
+- **`DSN`** is available in your project's **Settings** > **Projects** > **Project name** > **Client Keys
+  (DSN)** tab.
+
+</Step>
+
+<Step label="1.2">
+
+Go to the [Sentry API section](https://sentry.io/settings/account/api/auth-tokens/), and create an
+**auth token**. The token requires the scopes: `org:read`, `project:releases`, and
+`project:write`. Save them.
```

Comment:
Sentry has new organization auth tokens that are preferred for source maps uploaded as they are not bound to individuals but to org. 
```suggestion
Go to the [DEVELOPER SETTINGS -> Auth Tokens](https://sentry.io/settings/) and create a new token. The token is automatically scoped for Source Map Upload and Release Creation. Save it.
```